kratos/utilities/sensitivity_builder.h

```diff
@@ -147,6 +235,8 @@ class KRATOS_API(KRATOS_CORE) SensitivityBuilder
     ///@name Private Operations
     ///@{
 
+    static TSensitivityVariables GetVariableLists(const std::vector<std::string>& rVariableNames);
```

Comment:
Only reason is, it does not require any data from `SensitivityBuilder`, other than that there is no advantage of being static since this is a `private static` :|